R/parameters.R

```diff
@@ -63,8 +63,10 @@ parameters_demographic <- function(region, mixing_matrix = "Zimbabwe") {
     p_SW <- 0.007 * 0.5 # 0.7% women (50%) 15-49 Laga et al - assume this holds down to age 12
   } else if (region == "sudkivu"){
     p_SW <- 0.03 * 0.5 # WHO press release
-  } else if (region %in% c("burundi","bujumbura")){
-    p_SW <- 0.028 * 0.5 # Laga et al 
+  } else if (region == "burundi"){
+    p_SW <- 0.028 * 0.5 # Laga et al
+  } else if (region == "bujumbura"){
+    p_SW <- 3852/(792503*0.5) ## key pops report, taken as median of Bujumbura Mairie in Figure 7 
```

Comment:
```suggestion

   p_poss_SW  <-  sum(N_ASW + N_CSW)/ sum(N_age)
   p_SW <- 3852 / (792503 * p_poss_SW)  ## key pops report, taken as median of Bujumbura Mairie in Figure 7 
```
This should be as a % of total pop right? we multiply by 0.5 above because the other figures are quoted as % of women engaging in SW
I think the suggestion above should make sure the total number of SW comes out at 3852 but please check!

Reply:
OK did some digging and I know why - we use the figure for Bujumbura Mairie for the SW prop but then the "total" bujumbura population also includes Isare. When I remove Isare this is all fine. 
According to the KP document there are less SWs in Isare so we might have overinflated the value here, but I'm not sure how big a deal that is given the uncertainty in the SW size anyway @lwhittles